Create newsletter_13_Apr2020.md (#1036)
alan-turing-institute/the-turing-way: communications/newsletters/newsletter_13_Apr2020.md

To build on their work, we invite your contribution regarding data licensing that will be extremely useful for enhancing our understaning of licenses in Data Research.

We are currently working on the new design of the book ([see this preview](https://deploy-preview-977--the-turing-way.netlify.com/welcome)), which aims to provide easier access to the different volumes of the book and their subsections.

![An alternative landing page that displays a new browsing options for new volumes in The Turing Way](images/preview-Apr2020.png)
*Screenshot from the new proposed design of The Turing Way Book, developed in PR [#977](https://github.com/alan-turing-institute/the-turing-way/pull/977)*

## Relevant resources

### Translation efforts in _The Turing Way_

Anna Hadjitofi has written [a guide to using the Transifex interface](https://github.com/alan-turing-institute/the-turing-way/blob/translation/translation_guide.md) that Tony Yang is currently coordinating for _The Turing Way_ translation efforts.
They recommend that it's more effective when 2 people work on the translation process together so that they can review each other's translations.

> Transifex will also allows you to raise issues and browse all the past translations.

Author: ```suggestion
Transifex will also allow you to raise issues and browse all the past translations.
```

As a complement to the Research Excellence Framework (REF) exercise run in the UK, the Hidden REF will highlight outputs that are not selected or eligible for REF.
Until the end of June, the research community can suggest and comment on categories for which submissions should be accepted.
[Latest suggestions](https://www.software.ac.uk/blog/2020-04-06-first-look-new-categories-hidden-ref) include training material such as _The Turing Way_ resources or carpentry lessons, data and metadata standards, colleagues working in professional services and others.
To keep up to date with the Hidden REF, follow the initiative [on Twitter](https://twitter.com/HiddenRef).

### Mindful activities and Cross-Stitch Carpentry

Are you looking for some non-digital time after spending the entire day working online and attending virtual meetings?
Check this [Carpentries-style lesson on cross stitching](https://sgibson91.github.io/cross-stitch-carpentry/index.html) that Sarah Gibson recently launched to get beginners started with this mindful hobby.

> You can also read [this post](https://sgibson91.github.io/cross-stitch-carpentry/motivation/index.html) where Sarah and Malvika give more reasons for computer programmers and _The Turing Way_ community members to take time for mindful activities that can have positive impact on your mental health and well-being.

Author: ```suggestion
You can also read [this post](https://sgibson91.github.io/cross-stitch-carpentry/motivation/index.html) where Sarah and Malvika give more reasons for computer programmers and _The Turing Way_ community members to take time for mindful activities that can have a positive impact on our mental health and well-being.
```

![A lesson with links to different chapters to learn cross stitching for beginners](images/cross-Apr2020.png)
*Screenshot from the Carpentry Style lesson on Cross Stitching*

## Tips & Tricks for new contributors

If you are new to GitHub or _The Turing Way_, or both, here are a few resources to orient yourself with our project repository:

* Start by reading our [contribution guideline](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md), that lists the different pathways for the new and existing contributors to interact with this project.
* [GitHub issues](https://github.com/alan-turing-institute/the-turing-way/issues) are opened by different contributors on the topics they are interested in discussing, fixing or working on in the future.

> * You can browse these issues by labels (see [contributing through GitHub section](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md#contributing-through-github) and participate in a conversation by adding your comment under the respective issues. 

Author: ```suggestion
* You can browse these issues by labels (see [contributing through GitHub section](https://github.com/alan-turing-institute/the-turing-way/blob/master/CONTRIBUTING.md#contributing-through-github)) and participate in a conversation by adding your comment under the respective issues. 
```

* We have [good first issues](https://github.com/alan-turing-institute/the-turing-way/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to help our new contributors start by contributing little because every little bit helps.
* You can open a new issue on a topic where you would like to gather feedback from others.
* You are invited to introduce yourself to others on [our Gitter channel](https://gitter.im/alan-turing-institute/the-turing-way), or reach out to Malvika, _The Turing Way's_ community manager, by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).
  We love to say hello 👋

## Acknowledgements and celebrations

### Highlights from the Collaboration Workshop 2020

Several members from The Turing Way community participated at the [Virtual Collaboration Workshop 2020](https://software.ac.uk/events/2019-11-20-collaborations-workshop-2020), that was organised by Rachael Ainsworth, Community Manager of the Software Sustainability Institute (SSI), from 31 March to 2 April 2020.
Recordings and blog posts will be published on [SSI homepage](https://software.ac.uk) soon.

Meanwhile, here are some notable mentions from the event:

> * Andrew J. Stewart in his keynote on Open Research discussed the existing challenges around reproducibility in Open research in psychology and ways we can improve them ([slides](https://drive.google.com/file/d/1sN8xdUYD9flusHxR0SfTFy7EpLaGiDAK/view?usp=sharing)).

Author: ```suggestion
* Andrew J. Stewart in his keynote on Open Research discussed the existing challenges around reproducibility in open research in psychology and ways we can improve them ([slides](https://drive.google.com/file/d/1sN8xdUYD9flusHxR0SfTFy7EpLaGiDAK/view?usp=sharing)).
```
